src/app/activities/[activityId]/edit/sections/SubmitSection.js

```diff
@@ -42,7 +42,7 @@ function FormBlock() {
         <Box sx={{display: "flex", alignItems: "center"}}>
           <Checkbox size="small" sx={{ml: -1}}/>
           <Typography variant="body2" color="text.secondary">
-            I accept the <Link href="#">Terms and Conditions</Link> and <Link href="#">Privacy Policy</Link>
+            I accept the <Link href="#">Terms and Conditions</Link> and <Link href="/privacy">Privacy Policy</Link>
```

Comment:
link for terms and conditions?

Reply:
it should be link on tems and conditions for providers. This page is not created now